main/commons/src/main/java/org/cryptomator/common/settings/VaultSettingsJsonAdapter.java

```diff
@@ -54,7 +54,7 @@ public VaultSettings read(JsonReader in) throws IOException {
 			switch (name) {
 				case "id" -> id = in.nextString();
 				case "path" -> path = in.nextString();
-				case "mountName" -> mountName = in.nextString();
+				case "mountName" -> displayName = in.nextString(); //YES, this is correct (legacy reasons)
```

Comment:
Not exactly beautiful, though. I'd propose this:
1. During write, write to property `displayName`
2. During read, read both: `displayName` and `mountName`. set display name to `displayName` except for when it is null or empty, then use `mountName`.

Yes, this creates legacy code, but we can get rid of it eventually. It isn't breaking anything.

Reply:
Changed with https://github.com/cryptomator/cryptomator/pull/1318/commits/c3141e3601c7d8991ae625698f72ad3e3183c05b to use the `mountName` key, if the `displayName` is null.